scripts/gitlab/publish-awss3.sh

```diff
@@ -19,24 +19,21 @@ echo ${filetest[*]}
 for DIR in "${filetest[@]}";
 do
   cd $DIR
-  if [[ $DIR == "*windows*" ]];
+  if [[ $DIR =~ "*windows*" ]];
```

Comment:
I'm not sure why this didn't work for me but I assume $DIR string should contain the word windows.
Then this worked for me:
`if [[ "something" =~ "eth" ]]; then echo yes; else echo no; fi`

Reply:
`=~ "windows"` it is